test/verify/machineslib.py

```diff
@@ -1404,6 +1409,29 @@ def installWithErrorTest(dialog):
                                       {"Operating System": "You need to select the most closely matching Operating System"})
 
         # try to CREATE few machines
+        virtInstallVersion = self.machine.execute("virt-install --version")
+        if virtInstallVersion >= "2.2.0":
```

Comment:
Same problem with versions here. Let's say virt-install will go up to version 2.10.0, 2.11.0 ..., which will falsely evaluate that 2.10.0 >= 2.2.0 as false.